src/encoder.rs

```diff
@@ -2473,10 +2482,11 @@ fn encode_partition_bottomup(
     (best_rd, Some(best_decision))
 }
 
-fn encode_partition_topdown(fi: &FrameInvariants, fs: &mut FrameState,
+fn encode_partition_topdown(fi: &mut FrameInvariants, fs: &mut FrameState,
```

Comment:
Actually, it is used at line 2642 of encoder.rs to store the previous frame motion vectors.
The FrameInvariant structure is probably not the right place for that but the idea is to access to this data between frames. I'm open for better alternatives.

Reply:
Ideally it would go in a per-group-of-pictures context... Probably this would require more thinking though ^^;

Reply:
FrameState would be the correct place for this information for now. It could also be stored alongside the reference picture, which is advantageous when we are searching multiple references.

Reply:
How to keep this information between pictures if it is saved in FrameState?
A new FrameState is created at each receive_packet call in api.rs and I do not see how to access to previous frame data.

Reply:
You'd have to move them out of FrameState at the end of the call, you'd only be writing the current MVs there. You could copy them into the fi of the next frame encode, or better yet just store them alongside references.

Reply:
I am not sure to understand.
Do you mean storing this information in FrameState and transferring it between frames through FrameInvariants?
I do not see as well how to store it alongside frame references.